certora/ArrangerConduit.spec

```diff
@@ -593,8 +593,8 @@ rule cancelFundRequest(uint256 fundRequestId) {
         && infoHashAfter == infoHashBefore,
         "unrelated request params not as before";
 
-    assert requestedFundsAfter == requestedFundsBefore - requestBefore.amountRequested, "cancelFundRequest did not decrease by amount";
-    assert totalRequestedFundsAfter == totalRequestedFundsBefore - requestBefore.amountRequested, "totalRequestedFunds did not decrease by amount";
+    assert requestedFundsAfter == requestedFundsBefore - requestBefore.amountRequested, "requestedFunds did not decrease by amountRequested";
+    assert totalRequestedFundsAfter == totalRequestedFundsBefore - requestBefore.amountRequested, "totalRequestedFunds did not decrease by amountRequested";
```

Comment:
Hmmm I'm a bit confused here, we use to put the name of the function we are evaluating.